web/templates/weekly_info/index.html.eex

```diff
@@ -24,7 +24,7 @@
           ) %>
         </td>
         <td><%= weekly_info.service %></td>
-        <%= if weekly_info.attending_meal == true do %>
+        <%= if weekly_info.attending_meal do %>
```

Comment:
Minor: I think I'd do something like

<%= weekly_info.attending_meal |> to_display %>

```view_file.ex
  def to_display(true), do: "Yes"
  def to_display(true), do: "No"
``` 
This seems like it could be useful elsewhere.